baker/siteRenderers.tsx

```diff
@@ -100,60 +101,11 @@ import { GdocDataInsight } from "../db/model/Gdoc/GdocDataInsight.js"
 export const renderToHtmlPage = (element: any) =>
     `<!doctype html>${ReactDOMServer.renderToStaticMarkup(element)}`
 
-export const renderChartsPage = async (
-    knex: KnexReadonlyTransaction,
-    explorerAdminServer: ExplorerAdminServer
-) => {
-    const explorers = await explorerAdminServer.getAllPublishedExplorers()
-
-    const chartItems = await knexRaw<ChartIndexItem>(
-        knex,
-        `-- sql
-        SELECT
-            c.id,
-            cc.slug,
-            cc.full->>"$.title" AS title,
-            cc.full->>"$.variantName" AS variantName
-        FROM charts c
-        JOIN chart_configs cc ON c.configId=cc.id
-        WHERE
-            c.isIndexable IS TRUE
-            AND c.publishedAt IS NOT NULL
-            AND cc.full->>"$.isPublished" = "true"
-    `
-    )
-
-    const chartTags = await knexRaw<{
-        chartId: number
-        tagId: number
-        tagName: string
-        tagParentId: number
-    }>(
-        knex,
-        `-- sql
-        SELECT ct.chartId, ct.tagId, t.name as tagName, t.parentId as tagParentId FROM chart_tags ct
-        JOIN charts c ON c.id=ct.chartId
-        JOIN tags t ON t.id=ct.tagId
-    `
-    )
-
-    for (const c of chartItems) {
-        c.tags = []
-    }
-
-    const chartsById = lodash.keyBy(chartItems, (c) => c.id)
-
-    for (const ct of chartTags) {
-        const c = chartsById[ct.chartId]
-        if (c) c.tags.push({ id: ct.tagId, name: ct.tagName })
-    }
-
+export const renderDataCatalogPage = async (knex: KnexReadonlyTransaction) => {
+    const { __rootId, ...flatTagGraph } = await getFlatTagGraph(knex)
```

Comment:
It's prefixed on the flat tag graph JSON as a way to make it especially clear it's not part of the rest of the object that is otherwise keyed by numbers. Not quite consistent with the internal-only convention, but still defensible I think. Remapping it here might slightly harm greppability for little gain so I'm inclined to leave it as is. 🙂 

We could have just had an `isRoot` boolean on every entry in the flat tag graph, but oh well!

Reply:
All good then, I wasn't aware of the context.